server/src/main/java/org/apache/uniffle/server/ShuffleServerMetrics.java

```diff
@@ -72,7 +72,7 @@ public class ShuffleServerMetrics {
   private static final String IN_FLUSH_BUFFER_SIZE = "in_flush_buffer_size";
   private static final String USED_BUFFER_SIZE = "used_buffer_size";
   private static final String READ_USED_BUFFER_SIZE = "read_used_buffer_size";
-  private static final String USED_DIRECT_MEMORY_SIZE = "used_direct_memory_size";
+  private static final String NETTY_USED_DIRECT_MEMORY_SIZE = "netty_used_direct_memory_size";
```

Comment:
This metrics also should be moved into netty metric class

Reply:
And this should be initialized in the netty metrics class rather than in `shuffleServer` class

Reply:
Grpc use Netty, too.

Reply:
Oh I missed this. Let reserve this. Sorry @lifeSo

Reply:
@zuston  so, I revert the code to early version.